chingu-voyages/v50-tier2-team-14
# Added CSS to Map and FilterState

# Description
Fixed bugs in the FilterByCategory.js and add CSS for the Map directory



## Main changes explained:
- Fixed issue on FilterByCategory.js. - Some restaurants do not belong to any categories(pizza, steak, burgers, desserts), errors came out in the console. 
- Added 3 fonts style on tailwind.config - Luckiest Guy, Amaranth, Tensor
- Fixed the CSS for drop-down menu for FilterState

## How to test:
1. check into current branch
2. do `npm install` and `npm start` to run this PR locally
3. go to localhost:3000
4. click on drop-down menu
5. click on the arrow image 


## Screenshots or videos of changes:
![Screenshot 2024-08-09 at 6 30 10 PM](https://github.com/user-attachments/assets/646ffd05-b777-42ea-8d65-cb0d1e6f54d4)
os if necessary
